packages/api-specification/generate-docs.js

```diff
@@ -52,12 +56,13 @@ const generateMethod = (methodObj, className) => {
   if (!isIgnored(methodObj)) {
     const methodData = methodObj.flags.isStatic ? ' (static)' : '';
 
-    let badgeString = `![${methodObj.name}](https://img.shields.io/badge/Electron-no_test_data-lightgrey.svg) ![${methodObj.name}](https://img.shields.io/badge/OpenFin-no_test_data-lightgrey.svg)`;
+    let badgeString = `![${methodObj.name}](https://img.shields.io/badge/Electron-no_test_data-lightgrey.svg) ![${methodObj.name}](https://img.shields.io/badge/OpenFin-no_test_data-lightgrey.svg) ![${methodObj.name}](https://img.shields.io/badge/Browser-no_test_data-lightgrey.svg)`;
     if (testMatrixJson && testMatrixJson[`ssf.${className}.${methodObj.name}`]) {
       const testResults = testMatrixJson[`ssf.${className}.${methodObj.name}`];
       const electronBadgeColor = testResults.electron.total > 0 ? getBadgeColor((testResults.electron.passed / testResults.electron.total) * 100) : 'lightgrey';
       const openfinBadgeColor = testResults.openfin.total > 0 ? getBadgeColor((testResults.openfin.passed / testResults.openfin.total) * 100) : 'lightgrey';
-      badgeString = `![${methodObj.name}](https://img.shields.io/badge/Electron-${testResults.electron.passed}%2F${testResults.electron.total}-${electronBadgeColor}.svg) ![${methodObj.name}](https://img.shields.io/badge/OpenFin-${testResults.openfin.passed}%2F${testResults.openfin.total}-${openfinBadgeColor}.svg)`;
+      const browserBadgeColor = testResults.browser.total > 0 ? getBadgeColor((testResults.browser.passed / testResults.browser.total) * 100) : 'lightgrey';
```

Comment:
This line is repeated (with browser/openfin/electron varying) several times - might be good to refactor into a function to keep it dry? Could also call it from `getShieldMarkdown` to further reduce repetition.